src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -4031,6 +4036,37 @@ public static Value adv1(
     return RuntimeLibrary.continueValue();
   }
 
+  private record EightBitZone(DoubleModifier mod, int offset, String color) {}
+
+  private static final Map<Integer, EightBitZone> EIGHT_BIT_ZONES =
+      Map.ofEntries(
+          Map.entry(
+              AdventurePool.FUNGUS_PLAINS, new EightBitZone(DoubleModifier.MEATDROP, 50, "blue")),
+          Map.entry(
+              AdventurePool.HEROS_FIELD, new EightBitZone(DoubleModifier.ITEMDROP, 0, "green")),
+          Map.entry(
+              AdventurePool.VANYAS_CASTLE,
+              new EightBitZone(DoubleModifier.INITIATIVE, 200, "black")),
+          Map.entry(
+              AdventurePool.MEGALO_CITY,
+              new EightBitZone(DoubleModifier.DAMAGE_ABSORPTION, 200, "red")));
```

Comment:
"blue"

Reply:
So many comments just to say that I'd swapped Fungus Plains and Megalo-city ;)